cmd/all-in-one/main.go

```diff
@@ -103,7 +100,10 @@ by default uses only in-memory database.`,
 				svc.MetricsFactory.Namespace(metrics.NSOptions{Name: "jaeger"}))
 			version.NewInfoMetrics(metricsFactory)
 
-			tracerCloser := initTracer(svc)
+			tracer, err := jtracer.New("query-service")
```

Comment:
This is the cause of [the failing test](https://github.com/jaegertracing/jaeger/actions/runs/5563637373/jobs/10162715011?pr=4574#step:11:1029).

```suggestion
			tracer, err := jtracer.New("jaeger-query")
```

Reply:
Ah, sorry! I missed that.